submissions/OlgaBarnikova/Popup/index.html

```diff
@@ -0,0 +1,157 @@
+                              <img src="popup-icons/documents.png" alt="" 
+                              class="popup_img" />
+                              <div>Documents</div>
+                            </a>
+                         </li>
+                         <li>
+                            <a href="#" class="popup_box_img">
+                              <img src="popup-icons/blogs.png" alt="" 
+                              class="popup_img" />
+                              <div>Blogs</div>
+                            </a>                            
+                         </li>
+                         <li>
+                            <a href="#" class="popup_box_img">
+                                <img src="popup-icons/apps.png" alt="" 
+                                class="popup_img">
+                                <div>Apps</div>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="popup_box_img">
+                              <img src="popup-icons/news.png" alt="" 
+                              class="popup_img" />
+                              <div>News</div>
+                            </a>
+                         </li>
+                         <li>
+                            <a href="#" class="popup_box_img">
+                              <img src="popup-icons/search.png" alt="" 
+                              class="popup_img" />
+                              <div>Search</div>
+                            </a>                            
+                         </li>
+                         <li>
+                            <a href="#" class="popup_box_img">
+                                <img src="popup-icons/promo.png" alt="" 
+                                class="popup_img">
+                                <div>Promo</div>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="popup_box_img">
+                              <img src="popup-icons/tanslation.png" alt="" 
+                              class="popup_img" />
+                              <div>Translation</div>
+                            </a>
+                         </li>
+                         <li>
+                            <a href="#" class="popup_box_img">
+                              <img src="popup-icons/videos.png" alt="" 
+                              class="popup_img" />
+                              <div>Videos</div>
+                            </a>                            
+                         </li>   
+                    </ul>
+                    <ul class="popup_more">
+                        <li>
+                        <input id="popup-more" type="checkbox" class="popup_more_input">
+                        <label for="popup-more" class="popup_more_icon">More</label>
```

Comment:
Add styles on focus state. Also, hide this button when the second part of the popup is open.